app/config/FrontendAppConfig.scala

```diff
@@ -74,4 +74,6 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
   val emailVerificationUrl: String  = configuration.get[String]("urls.emailVerificationUrl")
 
   val emailVerificationEnabled: Boolean = configuration.get[Boolean]("features.email-verification-enabled")
+
+  val coreValidationService: Service = configuration.get[Service]("microservice.services.core-validation")
```

Comment:
Are there  config PRs to override core-validation?

https://catalogue.tax.service.gov.uk/service/ioss-registration-frontend/config